timelock-corruption-detection/src/main/java/com/palantir/timelock/corruption/detection/CorruptionStatus.java

```diff
@@ -17,16 +17,26 @@
 package com.palantir.timelock.corruption.detection;
 
 public enum CorruptionStatus {
-    HEALTHY(false),
-    CORRUPTION(true);
+    HEALTHY(false, false),
+    DIVERGED_LEARNERS(true, false), // this is false for now
+    VALUE_LEARNED_WITHOUT_QUORUM(true, false),
+    ACCEPTED_VALUE_GREATER_THAN_LEARNED(true, false),
+    DEFINITIVE_CORRUPTION_DETECTED_BY_LOCAL(true, true),
+    DEFINITIVE_CORRUPTION_DETECTED_BY_REMOTE(true, true);
```

Comment:
The first 4 are one layer and the bottom 2 are another. I'd suggest having maybe a `CorruptionCheckStatus` that is one of `HEALTHY, DIVERGED_LEARNERS ...` and then a `CorruptionStatus` (this class) just has `HEALTHY` and the two `DEFINITIVE_CORRUPTION` ones